modules/cloud-router-connection/variables.tf

```diff
@@ -90,14 +109,13 @@ variable "secondary_connection_name" {
   default     = ""
 }
 variable "secondary_bandwidth" {
-  description = "Connection bandwidth in Mbps"
+  description = "Secondary Connection bandwidth in Mbps"
   type        = number
   default     = 50
 }
-variable "aside_sec_fcr_uuid" {
-  description = "Equinix-assigned Fabric Cloud Router identifier for Secondary Connection"
+variable "zside_vd_sec_uuid" {
```

Comment:
I think you can remove this. This module implies that two different cloud routers will connect to a single virtual device. This should return to the variable that it is overwriting.

Reply:
I have removed the second FCR variable.